<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Out</class>
 <widget class="QWidget" name="Main_Out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QLabel" name="carNum">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车牌图片：</string>
    </property>
   </widget>
   <widget class="QLabel" name="outTime_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>出场时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="carNum_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车牌号码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="outVideo_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>671</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="carNum_image">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="inTime_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>入场时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="stayTime_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>停车时长：</string>
    </property>
   </widget>
   <widget class="QLabel" name="price_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>收费：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="out_butt">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>480</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>放行</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>502</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>是否月卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carNum_butt">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>130</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>车牌识别</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outTime_edit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="carNum_edit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="inTime_edit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="stayTime_edit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
